mysql/util.go

```diff
@@ -171,13 +170,10 @@ func AppendLengthEncodedInteger(b []byte, n uint64) []byte {
 
 func RandomBuf(size int) []byte {
 	buf := make([]byte, size)
-	// When this project supports golang 1.20 as a minimum, then this mrand.New(...)
-	// line can be eliminated and the random number can be generated by simply
-	// calling mrand.Intn()
-	random := mrand.New(mrand.NewSource(time.Now().UTC().UnixNano()))
-	min, max := 30, 127
+	random := mrand.New(mrand.NewSource(int64(mrand.Uint64())))
+	min, max := int32(30), int32(127)
 	for i := 0; i < size; i++ {
-		buf[i] = byte(min + random.Intn(max-min))
```

Comment:
by checking the source code of `Intn` I think there's not big different with `Int31n`. We don't need to update this file?